### PR TITLE
Add Performance Bot Best Practice MXP014 - Microflow CRUD placement

### PR DESCRIPTION
**Pull request by mrseanryan**
note: this is MxAssist Client team ref = AA-3298

SP version 9.12

@MariaShaposhnikova @jaspervanderhoek

### CODE REVIEW

File: content/refguide/performance-best-practices.md [hunk 1]
```diff
@@ -206,3 +206,11 @@ A microflow with a repeatedly committed variable may cause longer-running transa
 
 Refactor the microflow so that the variable is only committed once in the flow.
 
+## 13 Try to place CRUD activities closer to the end of the microflow [MXP014] {#mxp014}
+
+A microflow with CRUD (CReate/Update/Delete) activities that are placed too near the start event, may cause the database to acquire locks or resources earlier than necessary.
```
Comment (dogancakmak) [inline content/refguide/performance-best-practices.md:211]: Suggestion:
CRUD (Create/Read/Update/Delete) instead of CRUD (CReate/Update/Delete)

Comment (mrseanryan) [reply content/refguide/performance-best-practices.md:211]: hm but Read is not relevant.

Comment (mrseanryan) [reply content/refguide/performance-best-practices.md]: (updated)

**Review by dogancakmak — APPROVED** [review]

**Comment by MariaShaposhnikova** [comment]
I've copied changes from this PR to https://github.com/mendix/docs/pull/4338. It is easier to copy changes than solve the conflict.

**Pull request closed without merging**